.k8s/K8s-Deployment.md

```diff
@@ -0,0 +1,57 @@
+# Kubernetes Deployment
+## Deployment
+### The Tool
+To work with a K8s cluster you need to install Kubectl, a tool for controlling your K8s deployment.
+[Kubectl installation guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+The `docker` command is the Docker Swarm equivalent to `kubectl`.
+
+By default the deploy-k8s.groovy deployment script will deploy to a local Minikube instance. To deploy to a remote
+Kubernetes cluster, you will need to add the the Docker registry environment variable `K8S_DOCKER_REGISTRY`. You
+will also need to change `kubectl`'s context to the remote one with `kubectl config use-context <remote-context>`
+
+### Before Deploying
+Kubernetes manifests do not support adding config files directly in the manifest via a file path. Instead we need to use
```

Comment:
✏️ Left some comments about the README in this PR https://github.com/connexta/ion-search/pull/75 that I think apply here.